BeatSpeedrun/Source/Extensions/BSMLExtensions.cs

```diff
@@ -79,8 +79,7 @@ public virtual void Initialize()
 
         public virtual void Dispose()
         {
-            if (GameplaySetup.IsSingletonAvailable && BSMLParser.IsSingletonAvailable)
-                GameplaySetup.instance.RemoveTab(TabName);
+            GameplaySetup.instance.RemoveTab(TabName);
```

Comment:
The following code may be safer.
```suggestion
            if(GameplaySetup.instance != null && BSMLParser.instance != null)
                GameplaySetup.instance.RemoveTab(TabName);
```